### ISSUE TITLE
List sandboxes for specific realm

### ISSUE THREAD

**Issue by sfelius** [SalesforceCommerceCloud/sfcc-ci#172]
Currently, when running `sandbox:list` a table with all sandboxes for all realms a user has access to is returned. This list can grow substantially when working with multiple customers. 

It would be nice if we could add a `--realm` parameter that would only return a table of sandboxes for the specified realm:

`sfcc-ci sandbox:list --realm bbsv`

**Comment by jlbruno** [comment]
I came here to post the same thing - as a partner, we get quite a big list of returned sandboxes and hard to parse out just what I care about.

**Comment by tobiaslohr** [comment]
@sfelius The sandbox API does not provide a filtering of sandboxes on realm level unfortunately. Hence we cannot provide a filtering easily as part of the CLI. However, you can pipe the result of `sfcc-ci sandbox:list` and `grep` by realm, like that:

```
$ sfcc-ci sandbox:list | grep 'bbsv'
```

**Comment by tobiaslohr** [comment]
Closing this as there is no Sandbox API support for getting sandboxes per realm and a workaround was proposed through https://github.com/SalesforceCommerceCloud/sfcc-ci/issues/172#issuecomment-849414738

Please comment or open a new issue, if you see other options.
